ogr/services/github/service.py

```diff
@@ -166,3 +166,10 @@ def project_create(self, repo: str, namespace: str = None) -> "GithubProject":
             service=self,
             github_repo=new_repo,
         )
+
+    @property
+    def hostname(self) -> str:
+        service_repo_url = parse_git_repo(self.instance_url)
+        hostname = service_repo_url.hostname
+        return hostname
+
```

Comment:
This code is identical to the one in the abstract class and will be inherited automatically, feel free to drop it.